Makefile

```diff
@@ -13,10 +13,16 @@ include Makefile.global
 
 all: extension
 
+#build columnar only
+columnar:
+	$(MAKE) -C src/backend/columnar all
```

Comment:
Above this line, add:

`mkdir -p $(citus_top_builddir)/src/backend/columnar/safeclib`

Reply:
Oh, looks like it would be better to copy the `ENSURE_SUBDIRS_EXIST` line from `distributed/Makefile`.